test/test_ops.py

```diff
@@ -955,7 +955,7 @@ def test_pts(self, asset):
                 frames, asset.get_frame_data_by_index(frame_index)
             )
 
-            assert pts_seconds == start_seconds
+            assert pts_seconds == pytest.approx(start_seconds)
```

Comment:
I'm surprised we need to use `approx` so just to be on the safe side I removed it and we're getting:

```
FAILED test/test_ops.py::TestAudioDecoderOps::test_next[asset0] - assert 0.5760000000000001 == 0.576
FAILED test/test_ops.py::TestAudioDecoderOps::test_next[asset1] - assert 0.9359999999999999 == 0.936
FAILED test/test_ops.py::TestAudioDecoderOps::test_pts[asset0] - assert 0.5760000000000001 == 0.576
FAILED test/test_ops.py::TestAudioDecoderOps::test_pts[asset1] - assert 0.9359999999999999 == 0.936
```

Which is probably OK. But I still find it surprising considering the implementation of  [av_q2d](https://ffmpeg.org/doxygen/7.0/rational_8h_source.html#l00104) is just

```
static inline double av_q2d(AVRational a){
     return a.num / (double) a.den;
 }

```